projects/step-frontend/src/lib/modules/execution/components/execution-selection-table/execution-selection-table.component.html

```diff
@@ -1,10 +1,10 @@
 <step-table
- #tableRef
- [dataSource]="dataSource"
- matSort
- matSortActive="startTime"
- matSortDirection="desc"
- [matSortDisableClear]="true"
+  #tableRef
+  [dataSource]="dataSource"
+  matSort
+  matSortActive="startTime"
+  matSortDirection="desc"
+  [matSortDisableClear]="true"
```

Comment:
I see many formatting changes in this PR, wasn't the code commited with prettier earlier?

Reply:
there was a merge where I skipped git hooks (because they were not passing). probably that's why they were formatting just now